README.md

```diff
@@ -1,4 +1,6 @@
-# Rummager
+# Search API
+
+**This is a fork of [rummager](https://github.com/alphagov/rummager), and is in a half-renamed state!  We will clean this up as the migration progresses.**
```

Comment:
You're probably aware of this but FYI the note above looks to be incorrect now ☝️

Reply:
I've changed the note